app/models/user.rb

```diff
@@ -16,12 +16,13 @@ class User < ActiveRecord::Base
     uniqueness: { case_sensitive: false }
 
   has_secure_password
-  validates :password, length: { minimum: 6 }
+  validates :password, length: { minimum: 6 }, allow_nil: true
```

Comment:
Maybe im dumb, but are we dictating password length and allowing for null passwords here?